src/Model/Page.php

```diff
@@ -240,7 +240,7 @@ public function getRouteName()
      */
     public function setPageAlias($pageAlias): void
     {
-        if ('_page_alias_' != substr($pageAlias, 0, 12)) {
+        if ($pageAlias !== null && '_page_alias_' != substr($pageAlias, 0, 12)) {
```

Comment:
ins't is better to simply cast to string?

Reply:
I don't know if it's better or not. It depends on whether you want to store NULL or '\_page\_alias\_' into the object's property.

That's something _you_ should decide, since you know the framework way better than me. :)